media/css/firefox/features/common.scss

```diff
@@ -186,14 +188,22 @@ main {
     p {
         a:link,
         a:visited {
-            color: $color-burnt-orange;
+            color: $color-link-blue;
             text-decoration: none;
-        }
 
-        a:hover,
-        a:active,
-        a:focus {
-            text-decoration: underline;
+            &:hover,
+            &:focus,
+            &:active {
+                color: $color-link-blue-dark;
+                text-decoration: underline;
+            }
+
+            // Shameful specificity hack. Need to clean this up later.
+            &.cta-link:hover,
+            &.cta-link:focus,
+            &.cta-link:active {
```

Comment:
I'm not proud of this but it was a quick fix and at least isn't as bad as `!important`.

I intend to make the new blue the base link color in Pebbles and will be able to tidy this up then, but I didn't want to make that broad a change in this PR.